docs/modules/ROOT/pages/how-tos/configuring/component-nudges.adoc

```diff
@@ -11,25 +11,31 @@ However, if you build an application with {ProductName} and make references betw
 By default, {ProductName} only looks for image digest references in Dockerfiles, Containerfiles, and YAML files. However, you can xref:./customizing-the-build.adoc[customize your build pipeline] to adjust that list of files with an annotation. This functionality is only available for push builds, so add the following annotation under `annotations` in your component's `/.tekton/<component-name>-push.yaml` file:
 `build.appstudio.openshift.io/build-nudge-files: "<comma-separated list of link:https://docs.renovatebot.com/string-pattern-matching/[regex or glob patterns]>"`
 
-The existing list of patterns is: ".\*Dockerfile.*, .\*.yaml, .*Containerfile.*"  
+The existing list of patterns is: ".\*Dockerfile.*, .\*.yaml, .*Containerfile.*"
 ====
 
 == In the UI
 
 .Procedure
 
-To define the relationships between components in an application, complete the following steps in the {ProductName} UI:
+To define the relationships between components in a workspace, complete the following steps in the {ProductName} UI:
 
-. Navigate to your application. 
-. On any tab, use the *Actions* drop-down menu to select *Define component relationships*. 
+[NOTE]
+====
+Applications with only one component do not have the option to define relationships in the UI. To achieve the same result for single-component applications, you can define the relationship in the CLI.
```

Comment:
presumably this will be fixed with https://issues.redhat.com/browse/KFLUXBUGS-1313. Should we caveat this to say currently or link the issue? 

@Chr1st1anSears, do you have any thoughts?

Reply:
Hmm that's a good question. For external users, I think it'd probably be most helpful just to add "currently," as you suggested.

Reply:
@lance, would you be able to make that change?